### ISSUE TITLE
[Druid] Talent clear casting procs from things should not trigger it

### ISSUE THREAD

**Issue by bettermooon** [dalaranwow/dalaran-wow#5042]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
-Any druid any where and at any time
-using nogger frogger potion or finding mineral proc ur clear casting talent

**Expected behaviour**: Tell us what should happen instead
 expected behaviour is nothing should happen

**Steps to reproduce the problem**:

1. drink noggerfrogger/ turn on and off find minerals
2. u will get a clear casting proc
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
That's a blizlike behaviour. This aura procced wrong very different things on retail.

Source:

http://www.wowhead.com/spell=16864/omen-of-clarity#comments:id=1099753

Oddly enough, *this actually procced for me when I changed spec* (with dual spec). =P

http://www.wowhead.com/spell=16864/omen-of-clarity#comments:id=867604
As people have mentioned, this (and other "on spell cast" procs) *will proc off of professions, talent switches, and other non-combat abilities*. It does this because everything in wow that has a cast time (and a few things that don't) is considered a spell, as an example, look up a craftable item on here, go to its info page, and then click on the name of the item (not the icon), and you'll end up on the page for the spell that creates that item.
